README.md

```diff
@@ -7,20 +7,14 @@
 This XBlock allows course authors to create and edit course content in Markdown
 and displays it as HTML.
 
-## Installation
-You may install the markdown-xblock using its setup.py, or if you prefer to use pip, run:
+## Installation with [Tutor](https://docs.tutor.edly.io)
+You may install the markdown-xblock to your Tutor environment by adding it to the `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
 
-```shell
-pip install markdown-xblock==1.3.0
 ```
-
-If you prefer to install directly from Git, run:
```

Comment:
Perhaps we want to retain this information? We could mention that `OPENEDX_EXTRA_PIP_REQUIREMENTS` also supports `git+https` entries (unless it actually does not, which would be news to me).

Reply:
Sure, I added it back.